packages/ndla-ui/src/ToggleLicenseBox/ToggleLicenseBox.jsx

```diff
@@ -18,41 +22,112 @@ class ToggleLicenseBox extends Component {
     this.state = {
       expanded: false,
     };
+
+    this.dialog = null;
+    this.buttonWrapper = null;
+    this.focusTrapInstance = null;
   }
 
-  toogleLicenseBox() {
-    this.setState({
-      expanded: !this.state.expanded,
+  componentDidMount() {
+    this.focusTrapInstance = createFocusTrap(this.dialog, {
+      onDeactivate: () => {
+        if (this.state.expanded) {
+          this.setState({
+            expanded: false,
+          });
+        }
+
+        noScroll(false);
+      },
+      clickOutsideDeactivates: true,
     });
+
+    const computedStyles = window.getComputedStyle(this.dialog);
+    this.isNarrowScreen =
+      computedStyles.getPropertyValue('position') === 'fixed';
+  }
+
+  toogleLicenseBox() {
+    this.setState(
+      {
+        expanded: !this.state.expanded,
+      },
+      () => {
+        if (this.state.expanded) {
+          if (!this.isNarrowScreen) {
+            jump(this.buttonWrapper, {
```

Comment:
Litt usikker på om jeg ser behovet for jump her. Synes det så litt rart ut mtp breadcrumben som kommer inn i masthead. Det er ikke jeg som bestemmer dette, så bare se på det som en kommentar 😛 .

Reply:
hva mener du med at breadcrumben kommer inn i masthead?

Reply:
Mulig z-index er med på å få det til å se litt rart ut. Hvis du ser i i artikkelmal sidene så ser du at en breadcrumb blir synlig i masthead når du scroller forbi den som ligger over artikkelen.

Reply:
Har kun testet det i designmanualen før (siden ToggleLicenseBox ikke var i bruk i frontend). har implementert og testet ToggleLicensebox i frontend nå, og ser at jump ikke har noe funksjon selv om jeg har fikset z-index. fjerner :)